generate-dependabot-config/pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[project]
+name = "generate-dependabot-config"
+version = "0.1.0"
+description = ""
+authors = [
+    {name = "Henning Sprang",email = "henning.sprang@greenbone.net"}
+]
+readme = "README.md"
+requires-python = "^3.12"
+
+packages = [
+    { include = "action" },
+]
+
+dependencies = [
+    "jinja2 (>=3.1.6,<4.0.0)"
+]
+
+[tool.poetry]
+
+[tool.poetry.group.dev.dependencies]
+autohooks-plugin-pylint = "^23.10.0"
```

Comment:
please use ruff, not pylint

Reply:
thanks - I used the same set of authooks as in the other python projects in the repository but will change to this if that is the new standard